ui/app/AppLayouts/Communities/popups/CreateChannelPopup.qml

```diff
@@ -605,6 +639,136 @@ StatusStackModal {
+                        onClicked: {
+                            root.currentIndex = 2;
+                        }
+                    }
+                }
+                Item {
+                    Layout.fillWidth: true
+                    Layout.preferredHeight: Math.min(144, parent.height)
+                    Layout.alignment: Qt.AlignBottom
+                    Layout.leftMargin: Style.current.padding
+                    Layout.rightMargin: Style.current.padding
+                    Loader {
+                        anchors.fill: parent
+                        sourceComponent: (root.permissionsModel.count > 0)
+                        ? permissionsView : noPermissionsView
+
+                        Component {
+                            id: permissionsView
+                            PermissionsSettingsPanel {
+                                anchors.fill: parent
+                                viewWidth: scrollView.availableWidth - 32
+                                initialPage.header: null
+                                initialPage.topPadding: 0
+                                initialPage.leftPadding: 0
+                                permissionsModel: root.permissionsModel
+                                assetsModel: root.assetsModel
+                                collectiblesModel: root.collectiblesModel
+                                channelsModel: [
+                                    {"key": root.chatId,
+                                     "text": root.channelName,
+                                     "icon": root.channelEmoji,
+                                     "emoji": root.channelEmoji,
+                                     "color": root.channelColor,
+                                     "colorId": root.channelColor}
+                                ]
+                                communityDetails: ListModel{}
+                                showChannelSelector: false
+                                onRemovePermissionRequested: {
+                                    root.removePermissionRequested(root.chatId);
+                                }
+                                onUpdatePermissionRequested: {
+                                    root.updatePermissionRequested(root.chatId, permissionType, holdings, channels, isPrivate)
+                                }
+                            }
+                        }
+                        Component {
+                            id: noPermissionsView
+                            StatusBaseText {
+                                id: noPermissionsLabel
+                                horizontalAlignment: Text.AlignHCenter
+                                verticalAlignment: Text.AlignVCenter
+                                text: qsTr("No channel permissions")
+                            }
+                        }
+                    }
+                }
+            }
+        },
+        ColorPanel {
```

Comment:
Why is this here as a separate page? You are basically reverting my fixes from https://github.com/status-im/status-desktop/pull/12143; do you have some old/unmerged copy of this popup?

Reply:
I've replaced the `ColorPicker` with a `StatusPickerButton`, also we need the transitions and when it comes to permissions, we need  also a way to call `pushEditView(properties)` which is a function of the "second page" `PermissionsSettingsPanel` that loads the necessary component.